libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/helpers/RecyclerViewScrollPositionSaver.java

```diff
@@ -0,0 +1,10 @@
+package org.wordpress.android.util.helpers;
```

Comment:
I kind of wanted to enforce the feeling that `onSaveInstanceState()` and `onRestoreInstanceState()` lifecycle moments should be followed, and that seemed better expressed with an interface rather than using a plain object with similarly named methods